README.md

```diff
@@ -18,13 +18,13 @@ Keep it in mind that since `gir` is still under development, it generates warnin
 
 ## `gir` Modes
 
-There are two main modes of generation for `gir`; *FFI* and *API*.
+There are two main modes of generation for `gir`; _FFI_ and _API_.
```

Comment:
You don't like stars? :o

Reply:
This is a side-effect of me not turning my markdown formatter off in my editor.  I can clean these up if you'd prefer.